artiq/frontend/aqctl_corelog.py

```diff
@@ -79,8 +79,9 @@ async def get_logs(host):
                 name = 'firmware.' + m.group(2).replace('::', '.')
                 text = m.group(3)
                 log_with_name(name, level, text)
-    except:
-        pass
+    finally:
+        log_with_name("core log", logging.ERROR, "Terminated due to core device "\
+                      "connection failure")
```

Comment:
Or anything else that could raise an exception.

Just print the exception. And here might not be the best place to do it.